docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -113,6 +118,11 @@ The following table shows the inputs supported by the {agent} in {version}:
 |{y} (beta)
 |{y} (beta)
 
+|Osquery 
+|{n}
+|{y} (beta)
+|{y} (beta)
```

Comment:
I checked with the team, and for now, I think we should only list Osquery Manager support for `Fleet-managed Elastic Agent`. While technically aspects can work for standalone agents (e.g. you can manually schedule queries), currently they would not be able to do live queries. Similar to other items on this page, instead of listing it as `{n}`, we could note that it is `Under consideration`. What do you think, @mostlyjason?

Reply:
After thinking about it more, I suspect we should just list it as not supported for standalone mode, similar to Endpoint Security.

For standalone mode, people could use the Osquery Log Collection integration. But the newer Osquery Manager integration is made specifically for centrally managing osquery deployments in Fleet.

Reply:
Ok thanks I updated it so that it is only shown as support for Fleet-managed agents